Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+	"meta":
+	{
+		"title": "Bloom and HUD Texture Definitions",
+		"author": "iwubcode"
+	},
+	"groups":
+	[
+		{
+			"name": "Bloom",
+			"targets": [
+				{
+					"type": "efb",
+					"texture_filename": "efb1_n09_20x15_1"
+				},
+				{
+					"type": "efb",
+					"texture_filename": "efb1_n21_20x15_1"
+				}
+			]
+		},
+		{
+            "name": "DOF",
+            "targets": [
+                {
+                   "type": "efb",
+                    "texture_filename": "efb1_n10_320x240_4"
+                },
+                {
+                   "type": "efb",
+                    "texture_filename": "efb1_n11_320x240_1"
+                }
+            ]
+        }
+	],
+    "features":
+    [
+        {
+			"group": "DOF",
```

Comment:
We should move this to a "Native Resolution DOF" all target like we have for bloom.  See https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Native%20Resolution%20Bloom/all.txt and https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Native%20Resolution%20Bloom/metadata.json

Reply:
Do you feel comfortable making that change?

Reply:
I'd also suggest adding a "DOF Removal" like we do for bloom:  https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Bloom%20Removal/metadata.json .